src/postgres/tree/PostgreSQLTableTreeItem.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as _ from 'underscore';
+import * as vscode from 'vscode';
+import { AzureTreeItem } from 'vscode-azureextensionui';
+import { getThemeAgnosticIconPath } from '../../constants';
+import { getDocumentTreeItemLabel } from '../../utils/vscodeUtils';
+import { IPostgreSQLTreeRoot } from './IPostgreSQLTreeRoot';
+import { PostgreSQLSchemaTreeItem } from './PostgreSQLSchemaTreeItem';
+
+export class IPostgresTable {
+    public _id: string;
+    constructor(id) {
+        this._id = id;
+    }
+}
+
+export class PostgreSQLTableTreeItem extends AzureTreeItem<IPostgreSQLTreeRoot> {
+    public static contextValue: string = "PostgresTable";
+    public readonly contextValue: string = PostgreSQLTableTreeItem.contextValue;
+    // public readonly commandId: string = 'cosmosDB.openDocument';
+    public document: IPostgresTable;
+    public readonly parent: PostgreSQLSchemaTreeItem;
+
+    private _label;
+
+    constructor(parent: PostgreSQLSchemaTreeItem, document: IPostgresTable) {
```

Comment:
I'm confused by the naming of table/document in this file. Please clean that up

Reply:
Table for postgres is similar to the document for the cosmosdb account types, so the functionality we use for documents works for tables in postgres. However, If we use "Table" instead of "Document", it wont align with some methods we are using...such as "getDocumentTreeItemLabel".  Should we keep using "Document" instead of "Table" to keep things consistent across other types or use "Table"?

Reply:
Same thing happens with "Server" which is similar to the "DatabaseAccount/Account" in other types.

Reply:
I understand there will be some confusion around the names - just try your best. But also - make sure the stuff you're sharing actually makes sense. For example, does `getDocumentTreeItemLabel` still make sense to use for Postgres? I'm guessing it might be pretty specific to CosmosDB

I would've guess Table was more analogous to Collection. And a document was analogous to a row in the table